src/hotspot/share/gc/g1/g1CollectedHeap.hpp

```diff
@@ -178,6 +179,8 @@ class G1CollectedHeap : public CollectedHeap {
   // The block offset table for the G1 heap.
   G1BlockOffsetTable* _bot;
 
+  volatile size_t _live;
```

Comment:
I'm not happy with naming this `_live`, better use `_live_estimate`. The contents are not continuously updated and basically out of date after the first following allocation.
This includes the naming in all other instances too.

Reply:
I see your point - but that would probably lead to renaming `live()` method to `live_estimate()` (to keep the variable and the accessor method in sync) and that would break the nice symmetry we have now with `free()`, `used()` and `live()`. 
I have no strong feelings about this and if we can get quorum on this change I will do the renaming pass.